.github/workflows/stale.yml

```diff
@@ -0,0 +1,19 @@
+name: Mark stale issues and pull requests
+
+on:
+  schedule:
+  - cron: "0 0 * * *"
+
+jobs:
+  stale:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/stale@v1
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        stale-issue-message: 'Stale issue message'
```

Comment:
Shouldnt this be a actual message?

Reply:
you clicked this button didnt you
![image](https://user-images.githubusercontent.com/6224136/78028864-93f6de80-7357-11ea-9f85-2d85bdf133b1.png)